AllReadyApp/Web-App/AllReady/Areas/Admin/Controllers/EventManagerInviteController.cs

```diff
@@ -0,0 +1,114 @@
+        {
+            _mediator = mediator;
+            _userManager = userManager;
+        }
+
+        // GET: Admin/Invite/Send/5
+        [Route("[area]/[controller]/[action]/{eventId:int}")]
+        public async Task<IActionResult> Send(int eventId)
+        {
+            var @event = await _mediator.SendAsync(new EventByEventIdQuery { EventId = eventId });
+            if (@event == null) return NotFound();
+
+            if (!User.IsOrganizationAdmin(@event.Campaign.ManagingOrganizationId))
+            {
+                return Unauthorized();
+            }
+
+            return View("Send", new EventManagerInviteViewModel
+            {
+                EventId = eventId,
+                EventName = @event.Name,
+                CampaignId = @event.CampaignId,
+                CampaignName = @event.Campaign.Name,
+            });
+        }
+
+        // POST: Admin/Invite/Send/5
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        [Route("[area]/[controller]/[action]/{eventId:int}")]
+        public async Task<IActionResult> Send(int eventId, EventManagerInviteViewModel invite)
+        {
+            if (invite == null)
+            {
+                return BadRequest();
+            }
+
+            if (ModelState.IsValid)
+            {
+                var @event = await _mediator.SendAsync(new EventByEventIdQuery { EventId = eventId });
+                if (@event == null)
+                {
+                    return BadRequest();
+                }
+
+                if (!User.IsOrganizationAdmin(@event.Campaign.ManagingOrganizationId))
+                {
+                    return Unauthorized();
+                }
+
+                if (await _mediator.SendAsync(new UserHasEventManagerInviteQuery { EventId = invite.EventId, InviteeEmail = invite.InviteeEmailAddress }))
+                {
+                    ModelState.AddModelError("InviteeEmailAddress", $"An invite already exists for {invite.InviteeEmailAddress}.");
+                    return View("Send", invite);
+                }
+
+                var userToInvite = await _userManager.FindByEmailAsync(invite.InviteeEmailAddress);
+                if (userToInvite != null && userToInvite.ManagedEvents.Any(m => m.EventId == eventId))
+                {
+                    ModelState.AddModelError("InviteeEmailAddress", $"{invite.InviteeEmailAddress} is allready a manager for this event");
+                    return View("Send", invite);
+                }
+
+                invite.EventId = eventId;
+                var user = await _userManager.GetUserAsync(User);
+                await _mediator.SendAsync(new CreateEventManagerInviteCommand { Invite = invite, UserId = userToInvite?.Id });
+                // TODO: Send invite
+
+                return RedirectToAction(actionName: "Details", controllerName: "Event", routeValues: new { area = "Admin", id = invite.EventId });
+            }
```

Comment:
can you please remove `actionName:`, `controllerName:` and `routeValues:` from this RedirectToAction call? If you've added them anywhere else in the code, could you please remove them as well? thanks.